htdocs/app/view/login/Readme.md

```diff
@@ -0,0 +1 @@
+This folder contains the Login view and controller.
```

Comment:
I'd argue then that we either add those readmes regardless or delete all of them. All I care about is that it is consistent. :-)